src/test/cls_rbd/test_cls_rbd.cc

```diff
@@ -452,6 +452,34 @@ TEST_F(TestClsRbd, get_create_timestamp)
   ioctx.close();
 }
 
+TEST_F(TestClsRbd, get_access_timestamp)
```

Comment:
Nit: this test is removed in the next commit, so either squash the two commits or remove it from this one.